src/dotnet/APIView/APIViewWeb/Models/UsageSampleModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+using System.Security.Claims;
+
+namespace APIViewWeb
+{
+    public class UsageSampleModel
+    {
+        [JsonProperty("id")]
+        public string SampleId { get; set; } = IdHelper.GenerateId();
+        public string ReviewId { get; set; }
+        public int RevisionNum { get; set; }
```

Comment:
I don't see revision info list here if a sample has more than one revision. This class can be modified as below.

1. SampleId
2. ReviewId
3. List of UsageSampleRevisionModel